crates/uv/src/commands/build_frontend.rs

```diff
@@ -1155,39 +1167,55 @@ enum BuildMessage {
 }
 
 impl BuildMessage {
-    /// The filename of the wheel or source distribution.
-    fn filename(&self) -> &DistFilename {
+    /// The normalized filename of the wheel or source distribution.
+    fn normalized_filename(&self) -> &DistFilename {
         match self {
-            BuildMessage::Build { filename: name, .. } => name,
-            BuildMessage::List { filename: name, .. } => name,
+            BuildMessage::Build {
+                normalized_filename: name,
+                ..
+            } => name,
+            BuildMessage::List {
+                normalized_filename: name,
+                ..
+            } => name,
+        }
+    }
+
+    /// The filename of the wheel or source distribution before normalization.
+    fn raw_filename(&self) -> &String {
+        match self {
+            BuildMessage::Build {
+                raw_filename: name, ..
+            } => name,
+            BuildMessage::List {
+                raw_filename: name, ..
+            } => name,
         }
     }
 
     fn print(&self, printer: Printer) -> Result<()> {
         match self {
             BuildMessage::Build {
-                filename,
+                raw_filename: raw_name,
```

Comment:
Nit: this seems like a potentially unnecessary assignment since they're so similar (but not blocking obviously, up to you).